### DISCUSSION TITLE
Add URL parameters to Visibility Conditions

### DISCUSSION THREAD

**Discussion by SamP-dgt** [elementor/elementor#22795] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I would like to offer different views of the same posts (products/CPTs/articles...) for specific areas of the website, diverse audiences (not necessarily logged in), or on particular occasions.

### Describe the solution you'd like

It would be nice if you added  URL parameters option to your Visibility Conditions. That way I could assign different Elementor Templates to the same posts only by adding a specific (and custom) URL parameter, e.g.:

/?ver=alt
/?season=christmas
/?audience=journalists

... and the like.

The same would apply to custom Headers and Footers, part of Elementor Pro, as well as pop-ups.

### Describe alternatives you've considered

JetEngine offer this exact very feature in their JetThemeCore plugin. The same is available for Code Snippets (please see screenshots attached).

However, I'm not going to hard-code or even to install an entire plugin for a visibility condition. Moreover, Elementor Theme Builder is where I would love to manage my entire website, in terms of templates being used, without having to resort to other plugins. Please consider implementing it.

### Additional context

JetEngine JetThemeCore
<img width="732" alt="template-condition-editing-2" src="https://github.com/elementor/elementor/assets/125129165/87dcefd8-3ded-424d-96ec-0e7852f8f43c">
<img width="363" alt="header-and-footer-templates-from-the-library-2" src="https://github.com/elementor/elementor/assets/125129165/a806506d-bb43-4d16-be33-046568f3d03e">

Code Snippets
<img width="208" alt="Screenshot 2023-06-16 alle 11 00 15" src="https://github.com/elementor/elementor/assets/125129165/aa304e0d-d704-43b5-a24f-593c5294ca3d">
<img width="493" alt="Screenshot 2023-06-16 alle 11 00 30" src="https://github.com/elementor/elementor/assets/125129165/30ee34e6-17d7-4273-ab9a-87a3770caec9">

**Comment by gregbarbosa** [comment]
Adding on that we'd like to see this too.

Adding this feature would allow us to create a single page displaying different logos based on URL parameters. For example, someone arriving at the site with `utm_campaign=walmart` would see only the Walmart logo, while all other variations remain hidden.

Currently we rely on third-party plugins, but would prefer to keep things all in house with Elementor.

**Comment by nviitala** [comment]
Yes, this would be very important.
